plugins/meta/bandwidth/README.md

```diff
@@ -59,6 +59,6 @@ Both egressRate and egressBurst must be set in order to limit egress bandwidth.
 
 ## tc tbf documentation
 
-- [tldp traffic control](http://tldp.org/HOWTO/Traffic-Control-HOWTO/components.html)
+- [tldp traffic control](https://tldp.org/HOWTO/Traffic-Control-HOWTO/components.html)
```

Comment:
Amusingly this site has an incorrect certificate.  Perhaps we shouldn't change it?

Reply:
hmm, it can be accepted as long as the URL is not harmful.

Reply:
I have updated. Please check it!